contracts/utils/ExtendedSafeCast.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.6.4;
+pragma solidity ^0.6.12;
```

Comment:
I think we could use ranges for these contracts too (all utils), so others may import them